egs/librispeech/s5/RESULTS

```diff
@@ -443,6 +443,73 @@
 %WER 15.10 [ 7904 / 52343, 874 ins, 1070 del, 5960 sub ] exp/nnet3/tdnn_sp/decode_test_other_tgmed/wer_13_0.0
 %WER 16.29 [ 8528 / 52343, 828 ins, 1320 del, 6380 sub ] exp/nnet3/tdnn_sp/decode_test_other_tgsmall/wer_14_0.0
```

Comment:
This results file is really tough to read and compare. Would you be able to provide a brief set of important results right after you specify the command and present the detailed results after than ?